packages/block-library/src/navigation-link/editor.scss

```diff
@@ -1,17 +1,23 @@
 // Normalize Link and edit containers, to look mostly the same.
-.wp-block-navigation-link__field .components-text-control__input.components-text-control__input,
+.wp-block-navigation-link__field .components-text-control__input.components-text-control__input, // @todo: this selector doesn't appear to target anything.
```

Comment:
I could not find any instance of the `wp-block-navigation-link__field` except in this style.

Reply:
Same here. Should we just remove it now?

Reply:
👍 👍